README.rst

```diff
@@ -77,13 +77,22 @@ Or you can be specific about exactly which version you want::
 
     hashin "futures==2.1.3"
 
+You can also specify more than one package at a time::
+
+    hashin "futures==2.1.3" requests
+
 Suppose you don't have a ``requirements.txt`` right there in the same
-directory you can do this::
+directory you can specify ``--requirements-file``::
+
+    hashin futures --requirements-file stuff/requirements/prod.txt
```

Comment:
Perhaps I'm old-school but I think it always looks better to use a `=` with keyword arguments. Except if it's a single key one. E.g.
```
hashin futures --requirements-file=stuff/requirements/prod.txt
hashin futures --algorithm=sha512
```
I'm pretty sure argparse can cope with both but this style without the `=` looks so odd to me :)